hurricane/server/wsgi.py

```diff
@@ -46,7 +51,7 @@ def start_response(
             if hasattr(app_response, "close"):
                 app_response.close()  # type: ignore
         if not data:
-            raise Exception("WSGI app did not call start_response")
+            raise HurricaneWSGIException()
```

Comment:
If `HurricaneWSGIException` derives from `Exception`, please pass the message here the same way as before.